notebooks/ids_detection/IDS_using_LODA.ipynb

```diff
@@ -0,0 +1,507 @@
+{
```

Comment:
<p>you might want to merge this with the header below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/clx/pull/369/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>